### ISSUE TITLE
Log output of ShooterGame?

### ISSUE THREAD

**Issue by FezVrasta** [FezVrasta/ark-server-tools#42]
I think we could log the output of the ShooterGame session, so users can see the log and see what the server is currently displaying.
This will be particularly useful once we'll release 1.1 where screen was replaced by nohup

**Comment by TekThis** [comment]
This would be awesome and allow us to use log output to manipulate the server and/or it's webpage (depending on what the server outputs)

**Comment by FezVrasta** [comment]
I have tested this command:
```
nohup "$arkserverroot/$arkserverexec" "$arkserveropts" </dev/null >serverlog 2>&1 &
```
And it seems to print the entire output of the command inside the serverlog file

an example output is:

**Comment by Sispheor** [comment]
OK for me, be keep in mind that nohup must work even if we call the iit
script (I'am saying that because I 've removed screen for that reason)

2015-06-24 13:19 GMT+02:00 Fez Vrasta <notifications@github.com>:

> I have tested this command:
>
> nohup "$arkserverroot/$arkserverexec" "$arkserveropts" </dev/null >serverlog 2>&1 &
>
> And it seems to print the entire output of the command inside the
> serverlog file
>
> an example output is:
>
> [S_API FAIL] SteamAPI_Init() failed; SteamAPI_IsSteamRunning() failed.
> Setting breakpad minidump AppID = 346110
> Using binned.
> 4.5.1-0+UE4 7038 3077 402 3
> CtrlCHandler: Signal=15
> CtrlCHandler: Signal=15
>
> It could be used to detect unexpected crashes, in this way you know what
> the server printed before the crash.
>
> What do you think?
>
> ping @Sispheor <https://github.com/Sispheor>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FezVrasta/ark-server-tools/issues/42#issuecomment-114833393>
> .
>



-- 
*Nicolas MARCQ*

**Comment by FezVrasta** [comment]
Implemented in 1.2

by the way the nohup was not added by me, I'm not even sure if it's needed (I think no)

**Comment by klightspeed** [comment]
You're right - nohup isn't required in a non-interactive shell (i.e. when running a shell script).
